### PR TITLE
Fixed number of broken links in docs and comments

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

File: django/db/backends/oracle/operations.py [hunk 1]
```diff
@@ -242,7 +242,7 @@ def field_cast_sql(self, db_type, internal_type):
             return "%s"
 
     def last_executed_query(self, cursor, sql, params):
-        # http://cx-oracle.sourceforge.net/html/cursor.html#Cursor.statement
+        # http://cx-oracle.readthedocs.io/en/latest/cursor.html#Cursor.statement
```
Comment (timgraham) [inline django/db/backends/oracle/operations.py]: https

File: django/http/response.py [hunk 2]
```diff
@@ -240,7 +240,7 @@ def make_bytes(self, value):
         return force_bytes(value, self.charset)
 
     # These methods partially implement the file-like object interface.
-    # See http://docs.python.org/lib/bltin-file-objects.html
+    # See https://docs.python.org/2/library/stdtypes.html#file-objects
```
Comment (timgraham) [inline django/http/response.py]: the link for Python 3 might be https://docs.python.org/3/library/io.html#io.IOBase

File: docs/ref/contrib/gis/install/index.txt [hunk 3]
```diff
@@ -327,7 +327,7 @@ Fink
 of the `Fink`__ package system. `Different packages are available`__ (starting
 with "django-gis"), depending on which version of Python you want to use.
 
-__ http://schwehr.org/blog/
+__ http://schwehr.blogspot.com/
```
Comment (timgraham) [inline docs/ref/contrib/gis/install/index.txt]: https

File: docs/topics/auth/passwords.txt [hunk 4]
```diff
@@ -349,7 +349,7 @@ Include any other hashers that your site uses in this list.
 
 .. _sha1: https://en.wikipedia.org/wiki/SHA1
 .. _pbkdf2: https://en.wikipedia.org/wiki/PBKDF2
-.. _nist: http://csrc.nist.gov/publications/nistpubs/800-132/nist-sp800-132.pdf
+.. _nist: http://dx.doi.org/10.6028/NIST.SP.800-132
```
Comment (timgraham) [inline docs/topics/auth/passwords.txt]: https

File: docs/ref/django-admin.txt [hunk 5]
```diff
@@ -950,7 +950,7 @@ Python interpreter, use ``python`` as the interface name, like so::
    In older versions, use the ``--plain`` option instead of ``-i python``. This
    is deprecated and will be removed in Django 2.0.
 
-.. _IPython: http://ipython.scipy.org/
+.. _IPython: http://ipython.org/
```
Comment (timgraham) [inline docs/ref/django-admin.txt]: https

**Comment by scop** [comment]
Comments addressed, thanks.

**Pull request merged**